### ISSUE TITLE
[BUG] Windows Client: Getting a white screen background where the Valheim Plus logo usually is

### ISSUE THREAD

**Issue by SirFrostingham** [valheimPlus/ValheimPlus#796]
- **OS:** [e.g. Windows Client, Windows Server, or Linux Server] Windows Client
- **Valheim Game Version:** [e.g. 0.4.11, etc] latest
- **V+ Mod Version:** [e.g. 0.9, 0.9.2, etc.] latest

Windows Client: Getting a white screen background where the Valheim Plus logo usually is.

**To Reproduce**

Steps for the devs to reproduce the behavior:
1. Launch the game with just Valheim Plus mod
2. White background shows where Valheim Plus logo usually is

**Expected behavior**

Should show Valheim Plus logo.

**Troubleshooting**

What steps have you taken to try to fix? N/A

**Screenshots**

![image](https://user-images.githubusercontent.com/4725943/226707122-e7ff9fd5-0d78-49b9-a722-946e9af90115.png)

**Additional context**

None. I removed all mods and loaded the game with Valheim Plus mod only.

**Comment by PuszkaPotato** [comment]
Look at this PR #793

**Comment by SirFrostingham** [comment]
Tested. Still reproduces today.

**Comment by SSimp258** [comment]
> Tested. Still reproduces today.

Make sure you've updated to the latest Bepinex version of 5.4.2100.

**Comment by SirFrostingham** [comment]
I've been installing from here: https://github.com/valheimPlus/ValheimPlus/releases/latest

I install from whatever that is daily.

Just checked and it goes here: https://github.com/valheimPlus/ValheimPlus/releases/tag/0.9.9.11

**Comment by Grantapher** [comment]
Official release is broken with the new version of Valheim. You can use one of the community fixes or wait. So far, we've been waiting over two months with no idea when the owner will return.

See #793 for the current fixes.

**Comment by SirFrostingham** [comment]
Waiting. I have this built into a game launcher for clients that connect to my server. Sure I can add a step to pull BepInEx in the launcher, but I’ll just wait. No worries.

**Comment by As5kh** [comment]
I tried it #793 
I rent a G portal sever and after, I add this version seems to work ( display and everything ) but i didn't manage to server on my server due to an error of version :/

**Comment by SirFrostingham** [comment]
Ok. Scratch waiting... I went to [#793](https://github.com/valheimPlus/ValheimPlus/pull/793) and used the Thunderstore mod posted in the top "edit" post: https://valheim.thunderstore.io/package/Grantapher/ValheimPlus_Grantapher_Temporary/

Thank you very much for uploading that @Grantapher!

**Comment by Grantapher** [comment]
Glad it worked out.

**Comment by Grantapher** [comment]
@As5kh GPortal requires some extra steps, I recommend searching through [the discord](https://discord.gg/XamVGpgnJT), a bunch of people have got the fix working on GPortal.

**Comment by fbda** [comment]
> > Tested. Still reproduces today.
> 
> Make sure you've updated to the latest Bepinex version of 5.4.2100.

I using a ZAP Game Server. As an user i cannot update BepInEx, so when there is an official BepInEx (Valheim Plus) update for the Gaming Server Partner?

**Comment by ersan** [comment]
> > > Tested. Still reproduces today.
> > 
> > 
> > Make sure you've updated to the latest Bepinex version of 5.4.2100.
> 
> I using a ZAP Game Server. As an user i cannot update BepInEx, so when there is an official BepInEx (Valheim Plus) update for the Gaming Server Partner?

If they have no way of updating BepInExPack outside of Valheim Plus Official you need to find a new server host.

**Comment by KayHude** [comment]
> > > Tested. Still reproduces today.
> > 
> > 
> > Make sure you've updated to the latest Bepinex version of 5.4.2100.
> 
> I using a ZAP Game Server. As an user i cannot update BepInEx, so when there is an official BepInEx (Valheim Plus) update for the Gaming Server Partner?

I'd suggest to get in touch with ZAP and politely ask them how they want to continue with a possibly abandoned mod.

**Comment by SirFrostingham** [comment]
@Grantapher - Does your patch still work? with just ValheimPlus running, I'm now getting a server crash. I removed all mods but that one. Looks like there was a recent update? Thanks for your time.

**Comment by Grantapher** [comment]
It should be, make sure you go through [the install instructions](https://github.com/Grantapher/ValheimPlus/blob/grantapher-development/INSTALL.md) and then failing that join [the discord](https://discord.gg/XamVGpgnJT) and post your logs there.

**Comment by SirFrostingham** [comment]
It was working fine a few days ago, now there is a constant crash. *sad panda face* It may not be this temp mod.

Note I have custom automation and don’t use a launcher. I snag your mod off of thunderstore though (in my automation).

**Comment by SirFrostingham** [comment]
@Grantapher I checked the server crash logs and I had zero crashes before the 3/29 V+ temp mod update.

**Comment by SirFrostingham** [comment]
[crash.zip](https://github.com/valheimPlus/ValheimPlus/files/11128494/crash.zip)

Unity is crashing.

**Comment by SirFrostingham** [comment]
@Grantapher I checked that discord and others are stating your temp mod was working before the update.

**Comment by Grantapher** [comment]
Are you still using thunderstore? If so, is your mod up to date there? version ending in 13. (see compatibility [here](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.9.13))

Also, is it steam or xbox game pass version?

**Comment by SirFrostingham** [comment]
I am on a PC for both client and server. Reviewing server versions request.

**Comment by SirFrostingham** [comment]
@Grantapher 

![image](https://user-images.githubusercontent.com/4725943/229267233-f4fb4ef2-7b9f-4cb6-883c-ecf04a4eda09.png)

**Comment by Grantapher** [comment]
Make sure to check your BepInExPack_Valheim and Valheim game versions as well and match them up with [compatibility in the release](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.9.13)

**Comment by SirFrostingham** [comment]
Got it. My BepInEx does not match your intended target version, because I am still getting the packaged version of BepInEx that comes from V+ mod’s WindowsServer.zip release (example: https://github.com/valheimPlus/ValheimPlus/releases/tag/0.9.9.11).

I always liked how they packaged BepInEx with the release. I guess maybe I can backport to the prior version (before the last update) or automate the install of BepInEx.

Thanks for your time.

**Comment by Grantapher** [comment]
The zips attached to [the release](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.9.13) come with the correct version of bepinex, or did you use an old release?

**Comment by SirFrostingham** [comment]
Oh I see… I can get the zips from your page like I was already getting from V+ mod. This is a very simple change to my automation for me to fix. I was still installing V+ mod from the source and your thunderstorm package on top of that (which was overwriting the older V+ mod DLL plug-in). 

Thank you!

**Comment by Strength7828** [comment]
Deleted -  duplicate comment. for xbox game pass users check https://github.com/valheimPlus/ValheimPlus/issues/806

**Comment by Grantapher** [comment]
@SirFrostingham yep I tried my best to keep the same format of the official mod's releases.

**Comment by SirFrostingham** [comment]
Verified working server now with the updated BepInEx in place. Thanks for the help. I am very thankful you patched the need and keeping VaheimPlus mod alive.
